data/presets/traffic_sign/hazard/curves.json

```diff
@@ -0,0 +1,19 @@
+{
+    "fields": [
+        "curves",
+        "maxspeed/advisory"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "tags": {
+        "traffic_sign": "hazard",
+        "hazard": "curves"
+    },
+    "reference": {
+        "key": "hazard",
+        "value": "curves"
+    },
+    "name": "Curves Warning Sign"
```

Comment:
The MUTCD has two different signs that map to this tag and could be aliases: “reverse curve” and “winding road”.